os/logm/logm.c

```diff
@@ -88,6 +88,18 @@ int logm_internal(int flag, int indx, int priority, const char *fmt, va_list ap)
 		&& flag == LOGM_NORMAL && !up_interrupt_context()) {
 
 		flags = irqsave();
+#if (defined CONFIG_LOGM_SYNCHRONOUS) && (CONFIG_LOGM_TIMEOUT_MS != 0)
```

Comment:
please fix it. defined (CONFIG_LOGM_SYNCHRONOUS)

Reply:
Done.